packages/app/src/helpers/modernize/alternative_language.template

```diff
@@ -0,0 +1,15 @@
+msgid ""
+msgstr ""
+"Project-Id-Version: PACKAGE VERSION\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: ###CREATION_DATE###\n"
+"PO-Revision-Date: ###REVISION_DATE###\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"Language-Team: LANGUAGE <LL@li.org>\n"
```

Comment:
This seems wrong, should we just remove it?

Reply:
@amcgee pretty sure that is needed, since is part of the PO spec. That it is wrong ,is a totally different problem I think.

Reply:
@jason-p-pickering got it, thanks.  @Mohammer5 do we know where this li.org email came from?

Reply:
I don’t know much about the PO spec or the history here, so it’s possible this is fine as-is

Reply:
Looks like these are the default values:
https://www.gnu.org/software/trans-coord/manual/gnun/html_node/PO-Header.html

I think we could omit all the others and let these be handled manually:

```
"Report-Msgid-Bugs-To: \n"
"Language: ###LANGUAGE###\n"
"X-Generator: Translate Toolkit 2.2.5\n"
"X-Pootle-Path: ###POOTLE_PATH###\n"
"X-Pootle-Revision: 0\n"
```

<hr />

I'll add an argument for the `LANGUAGE` one, which will default to `LANGUAGE <LL@li.org>`

Reply:
`li.org` doesn't resolve, does the `LL@li.org` exist?  This header is suppose to specify "the mailing list on which the translation team can be reached" - seems like maybe we should put a DHIS2 email in there if anything.

In any case, since these files are actually parsed by our own tools and only used internal to the repo (and soon in transifex) I don't think we need most of these headers, unless we're trying to be backwards-compatible with Pootle.  I haven't found any references to these headers in the [Transifex](https://docs.transifex.com/formats/gettext) or [Gettext](https://www.gnu.org/software/gettext/manual/gettext.html#PO-Files) docs.

I vote either for removing these lines or leaving them as-is (wrong, but not super harmful) rather than adding options for them.